eduvpn/nm.py

```diff
@@ -44,6 +44,73 @@ def get_mainloop():
     return GLib.MainLoop()
 
 
+def get_iface() -> Optional[str]:
+    """
+    Get the interface as a string if there is a master device
+    """
+    client = get_client()
+    active_connection = client.get_primary_connection()
+    if active_connection is None:
```

Comment:
i would do `if not active_connection` everywhere where you compare to `is None`

Reply:
everywhere where I check for is None I only expect none and not other falsy types such as empty lists. But I will change it :+1:

Reply:
that is a good reason if things are possibly `0` or `[]`, but in this case not required.